integration_tests/jenkins_jobs/03_setting_distribution.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+cd $HOME/friday/integration_tests && \
+pip3 install -r ./requirements.txt && \
+PATH=$PATH:$HOME/go/bin && \
+python3 config_setting.py
+
+scp -i ~/ci_nodes.pem ~/.nodef/config/genesis.json opc@132.145.83.49:~/.nodef/config
+scp -i ~/ci_nodes.pem ~/config.toml opc@132.145.83.49:~/.nodef/config
+
+scp -i ~/ci_nodes.pem ~/.nodef/config/genesis.json opc@150.136.172.164:~/.nodef/config
+scp -i ~/ci_nodes.pem ~/config.toml opc@150.136.172.164:~/.nodef/config
+
+scp -i ~/ci_nodes.pem ~/.nodef/config/genesis.json ubuntu@140.238.73.77:~/.nodef/config
+scp -i ~/ci_nodes.pem ~/config.toml ubuntu@140.238.73.77:~/.nodef/config
```

Comment:
Need to copy /.nodef/config/contracts

Reply:
`make install` is executed in each node, and it contains copying contracts under `.nodef/contracts`

Reply:
The build information must be the same wasm file. In the current master version ...